hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestObjectPut.java

```diff
@@ -288,4 +298,82 @@ public void testEmptyStorageType() throws IOException, OS3Exception {
     //default type is set
     Assert.assertEquals(ReplicationType.RATIS, key.getReplicationType());
   }
+
+  @Test
+  public void testDirectoryCreation() throws IOException,
+      OS3Exception {
+    // GIVEN
+    final String bucketname = "bucketName";
```

Comment:
Can we use the `BucketName` field in this class instead? Having a local variable `bucketname` and class level `bucketName` is confusing.

Reply:
That is a possibility. I just want to keep unit tests isolated as much as possible for a piece of mind. Will refactor to that.